### DISCUSSION TITLE
Do `ret_pr0a.ogf` or `ret_pr0b.ogf` exist?

### DISCUSSION THREAD

**Discussion by GravisZro** [DescentDevelopers/Descent3#433] [Q&A] [answered]
So was going over the loading process and upon loading a level I noticed this part of the logging output:
```
Initializing cockpit.
Unable to find bitmap ret_pr0a.ogf
Unable to load ret_pr0a.ogf reticle image.
Unable to find bitmap ret_pr0b.ogf
Unable to load ret_pr0b.ogf reticle image.
Unable to find bitmap ret_pr0a.ogf
Unable to load ret_pr0a.ogf reticle image.
Unable to find bitmap ret_pr0b.ogf
Unable to load ret_pr0b.ogf reticle image.
Unable to find bitmap ret_pr0a.ogf
Unable to load ret_pr0a.ogf reticle image.
Unable to find bitmap ret_pr0b.ogf
Unable to load ret_pr0b.ogf reticle image.
```

Obviously it cannot find `ret_pr0a.ogf` or `ret_pr0b.ogf` and I checked all(?) the D3 HOG files but can't locate them.

These are hard coded values for the default reticle (part of the HUD). Do these files actually exist somewhere or can I remove them from the source code?

**Comment by Jayman2000** [comment, marked as answer]
I wasn’t able to find a copy of either of those files in the full version of Descent 3 or in Descent 3 Demo 2, but `d3demo.hog` from [version 1.1.1 of the Descent 3 Demo 1](http://descent3.com/files/descent3demo/d3demo111.exe) does contain those files.
